### PR TITLE
render tests for pages list

### PR DESCRIPTION
**Pull request by bluwave**
refs: MBL-13537
affects: Parent
release note: none

test plan
-----------------
did replace front page view tapGesture with a UIButton.  So also test student that able to to navigate to front page via pages.

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1015191&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Parent

[MBL-13537](https://instructure.atlassian.net/browse/MBL-13537)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.3% | 93.3% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Core/Core/Pages/PageList/PageListViewController.storyboard [hunk 1]
```diff
@@ -73,15 +73,25 @@
                                                         <userDefinedRuntimeAttribute type="string" keyPath="iconName" value="arrowOpenRightSolid"/>
                                                     </userDefinedRuntimeAttributes>
                                                 </imageView>
+                                                <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="7bY-hM-aJf">
+                                                    <rect key="frame" x="0.0" y="0.0" width="382" height="76"/>
+                                                    <connections>
+                                                        <action selector="frontPageTapped:" destination="JGx-7A-Apd" eventType="touchUpInside" id="ajK-L4-YhU"/>
```
Comment (vanwagonet) [inline Core/Core/Pages/PageList/PageListViewController.storyboard]: Should we use `primaryActionTriggered` instead of `touchUpInside`?

Comment (bluwave) [reply Core/Core/Pages/PageList/PageListViewController.storyboard]: I can switch it.  What would be the pros of doing this?

Comment (vanwagonet) [reply Core/Core/Pages/PageList/PageListViewController.storyboard]: I'm not positive. I'm pretty sure VO will send both. But it does sound more semantic, right?

Comment (armstrongnate) [reply Core/Core/Pages/PageList/PageListViewController.storyboard]: I prefer `primaryActionTriggered`. If we use it everywhere it saves us from having to remember what the primary action is for the different UIControls.

Comment (bluwave) [reply Core/Core/Pages/PageList/PageListViewController.storyboard]: You mean `valueChanged` and `touchUpInside`?

**Pull request merged**